### ISSUE TITLE
[Error] Circular Dependency while running task `:runner:jar`

### ISSUE THREAD

**Issue by dolphin2410** [dolphin2410/McSR#2]
Stacktrace
```
FAILURE: Build failed with an exception.

* What went wrong:
Circular dependency between the following tasks:
:runner:jar
\--- :runner:jar (*)

(*) - details omitted (listed previously)

```

**Comment by dolphin2410** [comment]
Fixed [#5bf521a](https://github.com/dolphin2410/McSR/commit/5bf521a)

**Comment by dolphin2410** [comment]
다시 발견됨..

**Comment by dolphin2410** [comment]
Fixed [#5893098](https://github.com/dolphin2410/McSR/commit/5893098)
